new_dawn_server/actions/models.py

```diff
@@ -2,25 +2,26 @@
 
 
 class UserToUserActionMetadata(models.Model):
-    user_account_from = models.ForeignKey("users.Account", on_delete=models.CASCADE, related_name="from_user",
-                                          null=True, blank=True)
-    user_account_to = models.ForeignKey("users.Account", on_delete=models.SET_NULL, related_name="to_user", null=True,
-                                        blank=True)
+    user_account_from = models.ForeignKey("users.Account",
```

Comment:
actually let's make from/to to be required. Otherwise it wouldn't be an action at all.

Reply:
Should the "to" to be nullable, since it sets to null when target user is deleted?

Reply:
hmm good question. Maybe just set on_delete to be cascade? I think an action with null source/target is not useful at all.